packages/core/test/lib/utils/traceData.test.ts

```diff
@@ -27,24 +19,24 @@ const mockedScope = {
   }),
 } as any;
 
-describe('getTracingMetaTags', () => {
-  it('returns the tracing tags from the span, if it is provided', () => {
+describe('getTraceData', () => {
+  it('returns the tracing data from the span, if a span is available', () => {
     {
-      vi.spyOn(SentryCore, 'getDynamicSamplingContextFromSpan').mockReturnValueOnce({
+      jest.spyOn(SentryCoreTracing, 'getDynamicSamplingContextFromSpan').mockReturnValueOnce({
         environment: 'production',
       });
 
-      const tags = getTracingMetaTags(mockedSpan, mockedScope, mockedClient);
+      const tags = getTraceData(mockedSpan, mockedScope, mockedClient);
 
       expect(tags).toEqual({
-        sentryTrace: '<meta name="sentry-trace" content="12345678901234567890123456789012-1234567890123456-1"/>',
-        baggage: '<meta name="baggage" content="sentry-environment=production"/>',
+        'sentry-trace': '12345678901234567890123456789012-1234567890123456-1',
+        baggage: 'sentry-environment=production',
       });
     }
   });
 
   it('returns propagationContext DSC data if no span is available', () => {
-    const tags = getTracingMetaTags(
+    const tags = getTraceData(
```

Comment:
l: since they are no longer tags

```suggestion
    const traceData = getTraceData(
```